mentorApp/.vscode/launch.json

```diff
@@ -2,30 +2,32 @@
     // Use IntelliSense to learn about possible attributes.
     // Hover to view descriptions of existing attributes.
     // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
-    "version": "0.2.0",
-    "configurations": [
+
+	"version": "0.2.0",
+	"configurations": [
 		{
 			"name": "Current Device",
 			"request": "launch",
 			"type": "dart"
 		},
 		{
-			"name": "Pixel 2",
+			"name": "Android",
 			"request": "launch",
 			"type": "dart",
-			"deviceId": "Pixel 2 API R"
+			"deviceId": "android"
 		},
 		{
-			"name": "iPhone 11 Pro Max",
+			"name": "iPhone",
 			"request": "launch",
 			"type": "dart",
-			"deviceId": "iPhone 11 Pro Max"
-		},
+			"deviceId": "iphone"
+		}
 	],
 	"compounds": [
 		{
 			"name": "All Devices",
-			"configurations": ["Pixel 2", "iPhone 11 Pro Max"]
+			"configurations": ["Android", "iPhone"],
 		}
 	]
+
```

Comment:
For more information, see: https://github.com/flutter/flutter/wiki/Multi-device-debugging-in-VS-Code